docs/user-tutorial/data-diagnosis.md

```diff
@@ -0,0 +1,144 @@
+---
+id: data-diagnosis
+---
+
+# Data Diagnosis
+
+## Introduction
+
+This tool is to filter the defective machines from thousands of benchmarking results automatically according to rules defined in **Rule file**.
```

Comment:
`This tool is to filter the defective machines from thousands of benchmarking results automatically` =>
`This tool is to filter the defective machines automatically from thousands of benchmarking results`

Reply:
updated